src/main/resources/scripts/anomalyDetectionTesting.txt

```diff
@@ -0,0 +1,79 @@
+##===========SETUP===============##
+sudo yum install -y java-17-amazon-corretto-devel
+java -version
+sudo yum -y update
+sudo yum -y install docker
+docker -v
+sudo service docker start
+sudo yum -y install git
+sudo yum remove -y python3.7
+sudo yum update -y
+sudo yum groupinstall "Development Tools" -y
+sudo yum erase openssl-devel -y
+sudo yum install openssl11 openssl11-devel  libffi-devel bzip2-devel wget -y
+wget https://www.python.org/ftp/python/3.9.9/Python-3.9.9.tgz
+tar -xf Python-3.9.9.tgz
+cd Python-3.9.9/
+./configure --enable-optimizations
+make -j $(nproc)
+sudo make altinstall
+sudo yum install -y python3-pip
+cd ~
+sudo update-alternatives --install /usr/bin/python3 python3 /usr/local/bin/python3.9 1
+
+##===========OpenSearch===============##
+git clone https://github.com/opensearch-project/OpenSearch.git
+cd OpenSearch
+git checkout feature/extensions
+./gradlew clean
+./gradlew publishToMavenLocal
+./gradlew localDistro
+##cd distribution/archives/linux-tar/build/install/opensearch-3.0.0-SNAPSHOT/
+##nohup ./bin/opensearch >/home/ec2-user/OpenSearchOutput.log 2>&1 &
+##curl -X GET localhost:9200
+cd ~
+
+##===========OpenSearch-SDK-Java===============##
+git clone https://github.com/opensearch-project/opensearch-sdk-java.git
+cd opensearch-sdk-java
+./gradlew clean
+./gradlew publishToMavenLocal
+cd ~
+
+##===========Anomaly Detection===============##
+git clone https://github.com/opensearch-project/anomaly-detection.git
+cd anomaly-detection
+./gradlew check
+nohup ./gradlew run >/home/ec2-user/AnomalyDetectionExtensionOutput.log 2>&1 &
```

Comment:
Do we have any script to stop the the nohup processes?

Reply:
I have to work on it. But as of now we will be terminating instance right after the execution via pipeline script. So it will kill all the process. Once I find the commands for it I will add it.

Reply:
We need to stop the nohup process as a part of this script. Otherwise they can become zombie proces